spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDFactory.java

```diff
@@ -80,7 +84,14 @@ public RDD<InternalRow> buildScanFromSQL(String sql) {
 
     TableInfo actualTable =
         bigQueryClient.materializeQueryToTable(
-            sql, options.getMaterializationExpirationTimeInMinutes());
+            sql,
+            options.getMaterializationExpirationTimeInMinutes(),
+            // Label to add to Query Job indicating that it was invoked as part of query pushdown
+            new HashMap<String, String>() {
+              {
```

Comment:
Should be replaced with `ImmutableMap.<String, String>builder().put(QUERY_JOB_LABEL, QUERY_PUSHDOWN_JOB_LABEL_VALUE).build()`

Reply:
done